### PR TITLE
Mips cleanups

### PR DESCRIPTION
**Pull request by bsdjhb**
This removes most of the MIPS-specific local patches I could find outside of share/mk and libexec/rtld-elf (which will be removed when merging future upstream commits).

One thing this doesn't remove is tools/tools/sycalls_timing which is built on all architectures.  There are some scripts in that dir that are very MIPS specific that might be candidates for removal unless they are still useful for future benchmarks?

### CODE REVIEW

File: contrib/jemalloc/include/jemalloc/jemalloc_FreeBSD.h [hunk 1]
```diff
@@ -75,7 +75,7 @@
 #endif
 #ifdef __mips__
 #ifdef __mips_n64
-#  define LG_VADDR		48
+#  define LG_VADDR		64
```
Comment (brooksdavis) [inline contrib/jemalloc/include/jemalloc/jemalloc_FreeBSD.h:78]: I'm pretty sure I upstreamed this one. I wonder what happened.

**Review by brooksdavis — COMMENTED** [review]
Need to remove usr/include/png from etc/mtree/BSD.usr.dist.

You probably want to kill libexec/rc/rc.d/de4_hdmi

There's a commented out line referencing de4ctl in tools/cheribsdbox/Makefile

File: tools/cheribsdbox/Makefile [hunk 2]
```diff
@@ -377,7 +377,6 @@ CRUNCH_ALIAS_nmtree= mtree
 # It seems like ${LIBNETBSD} works:
 CRUNCH_LIBS+=	${LIBNETBSD}
 
-# usr/sbin/de4ctl
 # usr/sbin/flashit
```
Comment (brooksdavis) [inline tools/cheribsdbox/Makefile:380]: flashit is also de4 specific and can go

Comment (bsdjhb) [reply tools/cheribsdbox/Makefile]: I axed it along with some other nearby lines that looked DE4-specific.

**Comment by arichardson** [comment]
This all looks reasonable to me. The tests for privilege-reduces ABIs and features like the per-function captable need to be rewritten as non-proof-of-concept code anyway if we want to support them.

**Review by brooksdavis — APPROVED** [review]

**Pull request merged**